### ISSUE TITLE
Do not draw geometry for every change, use boxes instead

### ISSUE THREAD

**Issue by Zverik** [ppawel/openstreetmap-website#14]
I suggest having two modes of operation:
1. In a regular history browsing mode geometry is not drawn (so there is no problem with a number of objects). Instead, rectangular grid should be used, but with fixed size, i.e. not "0.01 degree", but "50x50 pixels". This way it would be obvious where changes occured, and those boxes should be highlighed when hovering over changeset title (and vice-versa).
2. After clicking on a changeset, its geometry is displayed, as it does now, allowing to expand elements of the changeset, but not accidentally switch to another changeset. When the changeset is "closed", boxed view returns. This would probably call for separate "tab page" with "back" link, not mixing history and changeset details in a single one.

Right now there is just too much information displayed, regardless of individual changeset sizes. But at the same time, that abundance of information tells user absolutely nothing, except for "those objects were touched". Using boxes for history view would help differentiating changes, applying some heuristics or just making the picture understandable.

**Comment by Zverik** [comment]
This obviously would solve #8, #11, #12, also partly #13. #4 would be easier to implement with a separate tab for a changeset geometry.

**Comment by ppawel** [comment]
Yes, OWL API provides a lot of information and there is still more to come like changeset complexity. The UI definitely needs some simplification or splitting into multiple "pages" or "modes".

Your suggestion sounds good and certainly solves performance problems. I'm not sure what to do now because I quite like the current UI that shows everything - perhaps there is a way to make it simpler and easier to use without resorting to boxes and navigating between two modes?

#15 should help with the usefulness of what is shown. #12 should help with performance.

Keep the thoughts coming - I'm sure we will reach some conclusion :)

**Comment by matkoniecz** [comment]
I am not sure how bb would exactly work, but it seems that at least I at least in my region vastly prefer current solution.
